website/blog/index.html

```diff
@@ -15,6 +15,15 @@
 -->
 
 <!-- TODO: Avoid boilerplate. -->
+<div class="blog-listing">
+  <div>
+    <a class="blog-title" href="testing.html">Lower Test Toil With Better Local Testing</a>
+    <span class="blog-date">February 21, 2024</span>
```

Comment:
nit: don't forget to change the date when ready :)

Reply:
Done.